apps/server/src/modules/user/saga/delete-user-calendar-data.step.ts

```diff
@@ -0,0 +1,79 @@
+import { Logger } from '@core/logger';
```

Comment:
since this is now triggered indepdently, it should not be part ot user module
At least a TODO here tocreate calendar module and move this saga step there